src/GeneExpressionTSnePlot.js

```diff
@@ -98,8 +98,9 @@ const GeneExpressionScatterPlot = (props) => {
   const {loading, resourcesUrl, errorMessage} = props                       // Overlay
   const colourSchema = [`#d4e4fb`,`#95adde`,`#6077bf`,`#1151D1`,`#35419b`,`#0e0573`]; // light blue to dark blue
   const colourSchemaLength = colourSchema.length; 
-  const dataScale = plotData.max==null? 0:plotData.max.toFixed(0).toString().length; // The digit before demical
-
+  
+  const plotEnable = (plotData.max==null);
```

Comment:
I'm going to be extra picky here and say that your variable name is a bit off. If `plotData.max == null` evaluates to `true`, it means that we don't have expression so we _disable_ the plot. Either change the comparison, or the variable names. It's important not to confuse future developers with ambiguous variable naming. 

Additionally, you should get in the habit of always using `===` rather than `==` in JavaScript. Read more about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Also, you don't need brackets. And another convention is that we don't use semicolons at the end of the lines, so you can remove those as well.